src/mock/handlers/mock.ts

```diff
@@ -0,0 +1,31 @@
+import { rest } from 'msw';
```

Comment:
I would maybe rename this to `example.ts`.  I think that just from the file path `mock/handlers/mock.ts`, it's unclear of what it is when compared to `mock/handlers/example.ts` -- I think it's more clear that this is either an example file or a handler that resolves other examples.  Maybe even `for-examples.ts` is clearer? Then the file path sort of reads like a description `mock/handler/for-examples.ts`, so it's mock handlers for examples.